ui/apps/everest/src/pages/db-cluster-details/cluster-overview/cards/db-details/basic-information/basic.tsx

```diff
@@ -0,0 +1,135 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { beautifyDbTypeName } from '@percona/utils';
+import { Messages } from '../../../cluster-overview.messages';
+import OverviewSection from '../../../overview-section';
+import { BasicInformationOverviewCardProps } from '../../card.types';
+import OverviewSectionRow from '../../../overview-section-row';
+import { useContext, useMemo, useState } from 'react';
+import { UpgradeDbVersionModal } from './upgrade-db-version-modal/upgrade-db-version-modal';
+import { useDbVersionsList } from 'components/cluster-form/db-version/useDbVersions';
+import { useUpdateDbClusterVersion } from 'hooks/api/db-cluster/useUpdateDbCluster';
+import { DbClusterContext } from '../../../../dbCluster.context';
+import { DbClusterStatus } from 'shared-types/dbCluster.types';
+import { useEffect } from 'react';
+
+export const BasicInformationSection = ({
+  loading,
+  type,
+  name,
+  namespace,
+  version,
+}: BasicInformationOverviewCardProps) => {
+  const [openEditModal, setOpenEditModal] = useState(false);
+  const handleCloseModal = () => {
+    setOpenEditModal(false);
+  };
+  const { dbCluster /*temporarilyIncreaseInterval*/ } =
```

Comment:
I left "temporarilyIncreasInterval" to not forget to discuss it. Should i left this method at all? Or not? Because in this case i didn't use it, but should i left it for future in dbCluster context? 
It is possible that we will never need it, does it make sense to leave this method now?

Reply:
let's keep it, sure